.github/workflows/integration-tests.yml

```diff
@@ -103,11 +103,13 @@ jobs:
         id: run-tests
         run: |
           cd tests
+          BLUECHI_VERSION=$(../build-scripts/version.sh)
```

Comment:
I am wondering if we can't just export the BlueChi version in the tmt prepare step:
https://github.com/eclipse-bluechi/bluechi/blob/main/tests/plans/container.fmf#L16-L20
Either as part of the setup script or as a dedicated one. Could you try this out? @ArtiomDivak 

The problem with only passing the version here is that it'll only work when run inside the GH CI.